Xamarin.Forms.Core.UnitTests/MultiBindingTests.cs

```diff
@@ -162,7 +149,8 @@ public void TestNestedMultiBindings()
 							$"{nameof(Element.BindingContext)}.{nameof(GroupViewModel.PardonAllSuspensions)}",
 							source: new RelativeBindingSource(RelativeBindingSourceMode.FindAncestor, typeof(StackLayout))),
 					},
-					Converter = new AnyTrueMultiConverter()
+					Converter = new AnyTrueMultiConverter(),
+					FallbackValue = false,
```

Comment:
the converter sends UnsetValue for the purpose of getting the fallback, but no fallback was defined